grand/simulation/antenna/generic.py

```diff
@@ -129,8 +138,15 @@ def irfft(q):
         def fftfreq(n, t):
             dt = (t[1] - t[0]).to_value('s')
             return numpy.fft.fftfreq(n, dt) * u.Hz
-
-        E = field.E.represent_as(CartesianRepresentation)
+	
+#        print("E not repr", field.E)
+        # LWP: Seems unnecessary at least in the tested case - field.E already in cartesian
```

Comment:
Indeed. My understanding is that now coordinates would always be Cartesian in the local GRAND frame by default. I.e. it is the user's responsibility to provide proper coordinates.